### PR TITLE
Change default behavior to automatically create missing Discord channels

### LINKED ISSUE #23: Creating channels should be true by default
First of all, thank you very much for this project. I just used it to migrate a Slack workspace yesterday and it would have been a real pain, if not outright impossible, if I didn't have this. While migrating, I ran into a few issues from which I have a few requests. To be able to discuss different issues separately, I've opened a few issues. I can help to create a PR with some of the changes if you agree with them.

## The issue

I moved a Slack workspace to a completely fresh Discord server and imagine most people using this program would do the same. The first time I ran the program, I ran into a problem when the script tried to post messages because I didn't supply the `--create` parameter. I think this parameter should be true by default or at the very least also documented more clearly on this repository's README.

### PR DESCRIPTION
**Pull request by richfromm**
Former option `--create` is now `--no-create`

Issue: 23

### CODE REVIEW

**Comment by richfromm** [comment]
Attn: @shmulvad, since you filed the issue

File: CHANGES.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 ## Current releases, on [this fork](https://github.com/richfromm/slack2discord)
 
+### 2.8 (in progress)
```
Comment (richfromm) [inline CHANGES.md:5]: I'm going to try to bundle a number of issues into this release, there are some (like this) that ought to be fairly easy to implement.

File: slack2discord/config.py [hunk 2]
```diff
@@ -50,8 +50,8 @@
 
     Slack and Discord channel names should **not** include the leading pound sign (#)
 
-    dest Discord channels must already exist. This script will not create channels, it will only
-    post to existing channels.
```
Comment (richfromm) [inline slack2discord/config.py:54]: Woops, that was no longer true even before this, as of the addition of the (now former) `--create` option.

**Comment by waocats** [comment]
Looks great! No complaints here 🙂

**Pull request merged**